### ISSUE TITLE
google plus url format error in layouts/partials/social.html?

### ISSUE THREAD

**Issue by XingxianDeng** [yoshiharuyamashita/blackburn#59]
google plus profile url like this:
https://plus.google.com/u/0/118046788804899957719

in `config.toml`, if googleplus item like this,
`googleplus = "u/0/102591860149128628370"` or `googleplus = "102591860149128628370"`, I will got an error like blow.

![image](https://user-images.githubusercontent.com/18435987/32646767-1af788fa-c629-11e7-90f1-e76c2449aade.png)

then, I found code like below, I think it is may be an error.
```html
<a class="pure-menu-link" href="https://plus.google.com/+{{ . }}" target="_blank"><i class="fa fa-google-plus-square fa-fw"></i>Google+</a>
```

**Comment by HorribleGeek** [comment]
@XingxianDeng

Nope! That is for multiple sign in. If you have another account you will see that url as u/1 .. U/2.....

**Comment by HorribleGeek** [comment]
Close this issue!
